pages/project/read/views/components/rops.jsx

```diff
@@ -105,17 +101,10 @@ export function Rops({ project = {}, ropsYears = [], url, today = new Date() })
   const thisYear = today.getFullYear();
   const latestYear = Math.max(...ropsYears);
 
-  const deadline = subMilliseconds(new Date(`${latestYear + 1}-02-01`), 1);
-
-  const showNextYear = isAfter(today, deadline);
-  let years = ropsYears;
-
-  if (showNextYear) {
-    years = [
-      latestYear + 1,
-      ...years
-    ];
-  }
```

Comment:
This code is basically all a no-op because it's always testing if today is after Feb 1st of the following year.